raiden-ts/src/path/actions.ts

```diff
@@ -20,3 +21,13 @@ export const pathFindFailed = createStandardAction('pathFindFailed').map(
 export const pfsListUpdated = createStandardAction('pfsListUpdated')<{
   pfsList: readonly Address[];
 }>();
+
+export const udcBalanceFetch = createStandardAction('udcBalanceFetch')<{}>();
+
+export const udcBalanceUpdate = createStandardAction('udcBalanceUpdate')<{
+  balance: BigNumberish;
```

Comment:
Please, avoid using `BigNumberish` in core. It's a very broad type which should be used only on public interface and immediately validated to some of the specific ones. Prefer some of the `*Int` ones, as may fit on this case.